lib/io/src/adc.rs

```diff
@@ -0,0 +1,4 @@
+// ADC trait used to abstract the ADC peripheral
```

Comment:
```suggestion
/// ADC trait used to abstract the ADC peripheral
```
Just need a triple `/` for Rust doc